app/features/transactions/transaction.ts

```diff
@@ -48,6 +48,21 @@ export type CashuReceiveSwapTransactionDetails = {
   totalFees: Money;
 };
 
+/**
+ * Additional details related to the transaction.
+ */
+export type CashuSendQuoteDestinationDetails =
```

Comment:
lets call this destination details and also get rid of quote word in this file.

I'd like the transaction to be a concept that doesn't have to know anything about quotes